docs/illink-options.md

```diff
@@ -24,7 +24,8 @@ which can use one of following values.
 - `all` - Keep all members in root assembly
 - `default` - Use entry point for applications and all members for libraries
 - `entrypoint` - Use assembly entry point as only root in the assembly
-- `visible` - Keep all members and types visible outside of root assembly
+- `library` - All assembly members and data needed for secondary trimming are retained
```

Comment:
Before reading the code it was pretty unclear to me what the library mode would do. I think we should add more detailed documentation for it.

Reply:
Updated the wording to be more descriptive.